packages/uswds-core/src/js/utils/test/select.spec.mjs

```diff
@@ -1,11 +1,19 @@
-const fs = require("fs");
-const assert = require("assert");
-const select = require("../select");
+import { readFileSync } from "fs";
+import { fileURLToPath } from "url";
+import path from "path";
+import assert from "assert";
+import select from "../select.mjs";
 
-const TEMPLATE = fs.readFileSync(`${__dirname  }/select.template.html`);
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
```

Comment:
These are declared twice. Is that intentional?

Reply:
@amyleadem for `__filename` and `__dirname`? If so, I added a comment above https://github.com/uswds/uswds/pull/5091#discussion_r1061589161.

Basically the original `__dirname` only works in CJS. To get the same functionality in ESM we need a few more lines of code.